### PR TITLE
docs(readme): grammar and typo fix

### PR DESCRIPTION
**Pull request by himanshu010**
self-explanatory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -589,10 +590,10 @@ If an object is passed as entry point the value might be a string, array of stri
 ```js
 module.exports = {
   entry: {
-    catalog: { 
-      import: './catalog.js', 
-    }
-  }
+    catalog: {
+      import: "./catalog.js",
+    },
```
Comment (anikethsaha) [inline README.md]: not needed.

File: README.md [hunk 2]
```diff
@@ -589,10 +590,10 @@ If an object is passed as entry point the value might be a string, array of stri
 ```js
 module.exports = {
   entry: {
-    catalog: { 
-      import: './catalog.js', 
-    }
-  }
+    catalog: {
+      import: "./catalog.js",
```
Comment (anikethsaha) [inline README.md]: not needed

Comment (anikethsaha) [reply README.md]: do for the rest as well

File: README.md (hunk 1 above)
Comment (anikethsaha) [reply README.md]: same do for other

**Review by anikethsaha — CHANGES_REQUESTED** [review]
lets keep it consistent with the webpack's docs repo. 

- revert all `*`  to `-` (as it was before)
- revert `"` to `'` for the configs where strings are being used and as it was before `'`
- remove `,` after closing curly brackets `}`

**Comment by himanshu010** [comment]
changed everything to the previous format @anikethsaha

**Review by anikethsaha — APPROVED** [review]

**Pull request merged**